db/migrate/20170711193304_remove_accepted_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAcceptedFromUsers < ActiveRecord::Migration
+  def change
+    remove_column :users, :accepted, :boolean
```

Comment:
We can look at the rollback issue. For other people, yes, only put the final versions of the migrations into the branch to check in.